core/app/c/[communitySlug]/settings/tokens/page.tsx

```diff
@@ -21,8 +24,9 @@ export default async function Page(props: { params: { communitySlug: string } })
 		return notFound();
 	}
 
-	const [stages, existingTokens] = await Promise.all([
+	const [stages, pubTypes, existingTokens] = await Promise.all([
```

Comment:
this is only getting the first 10 pubtypes, but we probably want the user to be able to select/deselect all the pub types?